src/app/test/test.component.ts

```diff
@@ -197,8 +203,8 @@ export class TestComponent implements OnInit {
     }
   }
 
-  uploadReport(event: any): void {
-    const file: File = event.target.files[0];
+  uploadReport(event: Event): void {
+    const file: File = (event.target as TargetWithFiles)?.files[0];
```

Comment:
I like your solution, but i think it would be more readable to use: HtmlInputElement as the target, look at [this](https://github.com/frankframework/frankframework/blob/7452e2d449bd91d2ab49b88fe50f903a4c4cf555/console/frontend/src/main/frontend/src/app/components/input-file-upload/input-file-upload.component.ts#L19C1-L20C1) solution for reference

Reply:
It's not entirely clear to me. Do you mean to change the Event type to HTMLInputElement, or the fyle tpe, or both/something else?

Reply:
```suggestion
    const file: File = (event.target as HtmlInputElement)?.files[0];
```

Reply:
This has been implemented